test/groovy/util/JenkinsLoggingRule.groovy

```diff
@@ -38,12 +38,12 @@ class JenkinsLoggingRule implements TestRule {
             @Override
             void evaluate() throws Throwable {
 
-                testInstance.helper.registerAllowedMethod("echo", [String.class], {
+                testInstance.helper.registerAllowedMethod("echo", [String], {
                     echoInput ->
                         log += "$echoInput \n"
                 })
 
-                Throwable caught
+                Throwable caught = null
```

Comment:
Is is implicitly initiallized with null. What was wrong with the old way? --> blurs focus of PR